xpad.c

```diff
@@ -645,12 +666,16 @@ static const u8 xboxone_rumbleend_init[] = {
 static const struct xboxone_init_packet xboxone_init_packets[] = {
 	XBOXONE_INIT_PKT(0x0e6f, 0x0165, xboxone_hori_ack_id),
 	XBOXONE_INIT_PKT(0x0f0d, 0x0067, xboxone_hori_ack_id),
+	XBOXONE_INIT_PKT(0x1430, 0x079b, xboxone_hori_ack_id),
 	XBOXONE_INIT_PKT(0x0000, 0x0000, xboxone_power_on),
+	XBOXONE_INIT_PKT(0x1430, 0x079b, xboxone_power_on),
```

Comment:
this is not needed. the line above applies to all xboxone pads

Reply:
Fixed on commit 1daa749. 
Line removed.